### PR TITLE
Typo fixes

### PR DESCRIPTION
**Pull request by Saif-Discord**
Fixed typos in the docs files, and a couple of brackets in the custom-commands file.

### CODE REVIEW

**Review by ltzmax — CHANGES_REQUESTED** [review]
while you're at this, could you also [change here](https://guide.melonbot.io/docs/utility/sticky/#syntax)? its using ` instead of > there.

**Comment by Saif-Discord** [comment]
> while you're at this, could you also [change here](https://guide.melonbot.io/docs/utility/sticky/#syntax)? its using ` instead of > there.

Done, along with another minor fix.

File: docs/utility/sticky/sticky.mdx [hunk 1]
```diff
@@ -7,7 +7,7 @@ Sticky messages to your channels.
 ​
 #### Syntax
 ```
-[p]sticky toggleheader <true_or_false`
+[p]sticky toggleheader <true_or_false>
 ```
 #### Description
 **Toggle the header for stickied messages in this channel.**
```
Comment (ltzmax) [inline docs/utility/sticky/sticky.mdx:13]: This as well to match everything else.
```suggestion
Toggle the header for stickied messages in this channel.
```

**Review by ltzmax — CHANGES_REQUESTED** [review]

**Review by ltzmax — APPROVED** [review]
Thanks. everything looks good. 
@japandotorg can merge when he got time and is around.

**Comment by japandotorg** [comment]
Tysm for the pull request, appreciate much!!

**Pull request merged**